config/src/main/resources/application-prod.yml

```
@@ -0,0 +1,26 @@
+  port: 8888
+
+eureka:
+  client:
+    register-with-eureka: true  # Eureka Server에 등록
+    fetch-registry: true  # Eureka Server에서 서비스 목록 가져오기
+    service-url:
+      defaultZone: ${PROD_EUREKA_SERVICE_URL}
+  instance:
+    hostname: ${EUREKA_INSTANCE_HOSTNAME}
+    prefer-ip-address: false
+    non-secure-port: 80
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)